### PR TITLE
Add new repository - openshift/cluster-api-provider-kubevirt

### PR DESCRIPTION
**Pull request by nirarg**
This is done by running "make new-repo"
Adding the following repository: https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
...
-    build_farm_ocp:
-      clusters:
-      - app.ci
-      - build01
-      - build02
-      - build03
-      - build04
-      - vsphere
-      namespaces:
-      - ocp
```
Comment (davidvossel) [inline core-services/prow/02_config/_plugins.yaml:279]: the changes in this _plugins.yaml file look a bit suspicious considering we're just trying to enable ci on the openshift/cluster-api-provider-kubevirt repo.

File: ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      memory: 200Mi
+tests:
+- as: unit-tests
+  commands: GOFLAGS="-mod=readonly" make test
+  container:
+    from: src
+- as: func-tests
+  commands: go mod tidy && go mod vendor && make functest
+  container:
+    from: bin
```
Comment (davidvossel) [inline ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml:22]: I don't think we'll be successful in running the functional tests like this. For now i think we can just execute the unit tests until we get a func test suite that can execute on openshift ci.

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (nirarg) [reply core-services/prow/02_config/_plugins.yaml]: Yes, also noticed to this change, created by the auto generator (make new-repo)
I removed this change

File: ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml (hunk 2 above)
Comment (nirarg) [reply ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml]: I agree
I removed the func tests, The func tests will be added later

**Review by nunnatsa — APPROVED** [review]

**Pull request merged**